types/seat/wlr_seat.c

```diff
@@ -282,6 +307,20 @@ struct wlr_seat_client *wlr_seat_client_for_wl_client(struct wlr_seat *wlr_seat,
 	return NULL;
 }
 
+struct wl_client *wl_client_for_wlr_seat_client(
```

Comment:
This is already `wlr_seat_client->client`

Reply:
Oh. That removes one thorn, I totally missed that. Thanks.

Reply:
(force pushed)